piJEE/piJEE-ejb/pom.xml

```diff
@@ -26,6 +26,17 @@
             <version>7.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
the thing with this, is that it introduces a lot of bugs (zip slip, DOS on many levels, code execution ..) . I find it weird that a vulnerability assessment tool is so vulnerable. I advice, if possible, to find another tool. If not i hope some disclaimers are put to warn that this should only be used in a development environment, then it would be removed after creating a final deliverable.
If it is possible to decouple the tool and the application i think we should aim for that, but having a tool that itself is vulnerable puts me in a lot of doubt about its quality as a vulnerability assessment tool!